index.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module "github-enterprise-server-versions" {
+  export function getCurrentVersions(): Promise<number[]>
+}
```

Comment:
does it need to declare a module here? Wouldn't this achieve the same thing?

```suggestion
export function getCurrentVersions(): Promise<number[]>
```